### PR TITLE
chore: upgrading to core 0.18 and api, removing metrics

### PR DESCRIPTION
**Pull request by obecny**

### CODE REVIEW

File: src/lightstep-opentelemetry-launcher-node.ts [hunk 1]
```diff
@@ -67,47 +62,20 @@ let fail: types.FailureHandler;
...
-        configureHostMetrics(config);
-        resolve();
-      }, reject);
-    });
-  };
-  sdk.start = start.bind(sdk);
-
   return sdk;
+
+  // return patchSDK(sdk, config);
```
Comment (mwear) [inline src/lightstep-opentelemetry-launcher-node.ts]: Can we delete this?

```suggestion
```

File: src/lightstep-opentelemetry-launcher-node.ts [hunk 2]
```diff
@@ -275,54 +236,13 @@ function configureBaseResource(
   const baseResource: Resource = new Resource(attributes);
 
   if (config.resource) {
-    config.resource = config.resource.merge(baseResource);
+    config.resource = baseResource.merge(config.resource);
+    // config.resource = config.resource.merge(baseResource);
```
Comment (mwear) [inline src/lightstep-opentelemetry-launcher-node.ts]: We can probably get rid of this. Was merge order reversed in OTel?
```suggestion
```

File: src/types.ts [hunk 3]
```diff
@@ -1,14 +1,17 @@
+import { DiagLogger, DiagLogLevel } from '@opentelemetry/api';
 import { NodeSDKConfiguration } from '@opentelemetry/sdk-node';
 
 /** Lightstep specific configuration options */
 export interface LightstepConfigType {
   accessToken?: string;
-  metricsHostEnabled?: boolean;
+  // metricsHostEnabled?: boolean;
```
Comment (mwear) [inline src/types.ts]: Are we keeping this for when we re-enable metrics, or should we remove it. If we want to remove it, we should remove `metricEndpoint` below.

**Review by mwear — COMMENTED** [review]
Mostly lgtm. There were a few comments that I think we can remove.

File: src/lightstep-opentelemetry-launcher-node.ts (hunk 2 above)
Comment (obecny) [reply src/lightstep-opentelemetry-launcher-node.ts]: cannot really find if that was the case, but the test was failing and it seems like the user config was overwritten with defaults

File: src/types.ts (hunk 3 above)
Comment (obecny) [reply src/types.ts]: at first I wanted to keep it, but the history is in git anyway, removing.

**Review by mwear — APPROVED** [review]

**Pull request merged**